dev-requirements.txt

```diff
@@ -1,9 +1,8 @@
-black==19.10b0 ; python_version >= '3.6'
+black==19.10b0
 pytest<6
 pytest-qt
-mock ; python_version >= '3'
-mock<4; python_version < '3'
-schemathesis==1.3.4 ; python_version >= '3'
+mock
```

Comment:
Is this required when we're using `unittest.mock`?

Reply:
It's in the py3 std lib.

Reply:
Forgot to remove this one!